apps/prairielearn/src/lib/editors.js

```diff
@@ -95,119 +96,125 @@ export class Editor {
     callback(null);
   }
 
-  doEdit(callback) {
+  assertCanEdit() {
+    // Do not allow users to edit without permission
+    if (!this.authz_data.has_course_permission_edit) {
+      throw error.make(403, 'Access denied (must be course editor)');
+    }
+
+    // Do not allow users to edit the exampleCourse
+    if (this.course.example_course) {
+      throw new Error(`Access denied (cannot edit the example course)`);
```

Comment:
Let's make this a 403 like the above error.

Reply:
Updated this to throw a 403 in [15e6a55](https://github.com/PrairieLearn/PrairieLearn/pull/9439/commits/15e6a551ace1b217ed76fec735c5306f682080d1).

Reply:
@mylesw27 the commented-on line wasn't actually changed in that commit, from what I can tell.

Reply:
Sorry about that! It is now _actually_ changed in [ea3d984](https://github.com/PrairieLearn/PrairieLearn/pull/9439/commits/ea3d984b36cb388f772a3f33dcda7bc4224a507c).